specification/cognitiveservices/data-plane/TranslatorText/stable/v3.0/TranslatorText.json

```diff
@@ -12,31 +12,23 @@
   },
   "securityDefinitions": {
-    "apim_key": {
+    "apiKeyHeader": {
```

Comment:
this is breaking change, need a new service tag. Just to double confirm, is this a swagger file bug fixing, or service behavior change?